### ISSUE TITLE
[Effect Suggestion] So close, yet so far.

### ISSUE THREAD

**Issue by anto4440** [gta-chaos-mod/ChaosModV#766]
**Name of effect**
So close, yet so far/Impossible challenge.

**Duration if timed effect (long or short timed?)**
Short

**Effect Description**
Creates a forcefield like normal, but around the players objective. If there are multiple objectives, they all get a forcefield. Yes, i am aware of the potential softlocks and times where this will have no effect, but i still think its worth it.

**Comment by Da-Best** [comment]
this could lead to a fail loop if they need to meet someone and the force field throws the character they need to meet away from the objective.

**Comment by anto4440** [comment]
Yes, i aknowledged that that was one of the risks this effect brought.

Den tor. 11. mar. 2021 kl. 17.12 skrev Da-Best ***@***.***>:
